pkg/operator/operator.go

```diff
@@ -495,6 +550,14 @@ func (o *Operator) ensureCerts(ctx context.Context, dir string) ([]byte, error)
 	return caData, nil
 }
 
+func fileExists(f string) bool {
+	_, err := os.Stat(f)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return true
```

Comment:
Is it true that the file exists when there's an error? Have you considered return `bool, error` that way you could early exit on the operator startup?

Reply:
I'll change this to take any error as a false for fileExists because I think the other possible errors would indicate the file isn't going to be readable.